config/routes.rb

```diff
@@ -209,6 +209,10 @@
     get "*path", to: "gone#show"
   end
 
+  constraints FullPathFormatRoutingConstraint.new("html_publication") do
```

Comment:
If it doesn't cause the thing to break, can you move this block above landing page? (I'm trying to keep things in alphabetical order as much as possible - obviously we have to have all the fixed routes, then all the FormatRoutingConstraints, then all the FullPathFormatRoutingConstraints, but within each group I'm trying to sort alphabetically as a second sort.